src/containers/quiz/quiz-question/Question.tsx

```diff
@@ -0,0 +1,175 @@
+}) => {
+  const { t } = useTranslation()
+
+  const { isMultipleChoice, isOpenAnswer } = determineQuestionType(
+    question.type
+  )
+
+  const ContainerComponent = appCardWrapper ? AppCard : Box
+
+  const isCorrect = isCorrectAnswer(question, value)
+
+  const iconStyles = styles.icon(
+    isCorrect ? AnswerStatusEnum.Correct : AnswerStatusEnum.Incorrect
+  )
+
+  const correctnessIcon =
+    showAnswersCorrectness &&
+    (isCorrect ? <CheckIcon sx={iconStyles} /> : <CloseIcon sx={iconStyles} />)
+
+  const shouldShowCorrectAnswers = showCorrectAnswers && !isOpenAnswer
+
+  const correctAnswersList =
+    shouldShowCorrectAnswers &&
+    question.answers
+      .filter((item) => item.isCorrect)
+      .map((item) => (
+        <Answer
+          checked
+          isCorrect={item.isCorrect}
+          isEditable={false}
+          key={item.text}
+          label={item.text}
+          showCorrectness
+          text={item.text}
+          type={question.type}
+        />
+      ))
+
+  const correctAnswers = shouldShowCorrectAnswers && (
+    <Box sx={styles.correctAnswers.root}>
+      <Typography sx={styles.correctAnswers.title}>
+        {t('myResourcesPage.quizzes.correctAnswers')}
+      </Typography>
+      <Box sx={styles.correctAnswers.list}>{correctAnswersList}</Box>
+    </Box>
+  )
+
+  const answersList = question.answers.map((answer) => {
+    const isChecked = Boolean(
+      isMultipleChoice ? value?.includes(answer.text) : value === answer.text
+    )
+
+    const handleChange = () => {
+      if (isMultipleChoice) {
+        const prev = (value as string[]) ?? []
+        const newValue = prev.includes(answer.text)
+          ? prev.filter((item) => item !== answer.text)
+          : [...prev, answer.text]
+
+        handleNonInputValueChange(newValue)
+      } else {
+        handleNonInputValueChange(answer.text)
+      }
+    }
+
+    return (
+      <Answer
+        checked={isChecked}
+        isCorrect={answer.isCorrect}
+        isEditable={isEditable}
+        key={answer.text}
+        label={answer.text}
+        onCheckboxChange={handleChange}
+        showCorrectness={showAnswersCorrectness}
+        text={answer.text}
+        type={question.type}
+        value={answer.text}
+      />
+    )
+  })
+
+  const multipleChoiceAnswersBlock = isMultipleChoice ? (
+    <FormGroup sx={styles.answersContainer}>{answersList}</FormGroup>
+  ) : (
+    <RadioGroup sx={styles.answersContainer}>{answersList}</RadioGroup>
+  )
+
+  const answersBlock = isOpenAnswer ? (
+    <Answer
+      isCorrect
+      isEditable={isEditable}
+      label={question.text}
+      onTextInputChange={handleInputChange}
+      showCorrectness={showAnswersCorrectness}
+      text={question.text}
+      type={question.type}
+      value={value as string}
+    />
+  ) : (
+    multipleChoiceAnswersBlock
+  )
+
+  const pointsBlock = showPoints && (
+    <Typography sx={styles.type}>{Number(isCorrect)}/1</Typography>
```

Comment:
What does it mean?

Reply:
That is a points block. Shown after quiz was finished and if corresponding settings were enabled. For now, we can't set points per question, so I every question costs one point. `isCorrect` is a boolean, I used `Number()` to convert it into number `0` or `1` to show scored points per question.